### PR TITLE
Added Profile Pic to the TEAM.md file

### PR DESCRIPTION
**Pull request by Akshay2996**
@proful @pranavgoel29 do review these changes & merge the files accordingly. Preview the changes before approving to see the changes made.

### CODE REVIEW

**Comment by proful** [comment]
@pranavgoel29 are you OK with these changes?

**Comment by proful** [comment]
Merging to avoid additional merge conflicts for @Akshay2996

@pranavgoel29 you can improve on top of it.

**Comment by Akshay2996** [comment]
@proful me & @pranavgoel29 had an offline call & we figured it out.

**Comment by pranavgoel29** [comment]
> @proful me & @pranavgoel29 had an offline call & we figured it out.

I just checked the changes Image is still squashed we have to work on it again.

File: docs/TEAM.md [hunk 1]
```diff
@@ -1,78 +1,77 @@
...
-| [krishnadevz](https://github.com/lokunade)         | I am krishna 4 years of expereince in web-development Current skills(HTML/CSS/JS/REACTJS/C/bootstrap4) from india currrently i am final year CS gradd student and Learning MERN✡ Stack Developer SelfTaught-Dev/Opensourcer🛠 #100DaysOfCode(R2) FOSS❤ I like to build things & Write things related to Web🌐⚙ #webdev 🐱‍💻 |
-| [Melissa Huerta](https://github.com/piratelicorne) | Hi! I'm Systems Engineer from Perú, 3+ yrs of experience in IT Infrastructure, I have knowledge in C and Javascript. I recently started in Development as Flutter Developer, currently working on side projects 👩🏽‍💻 and blogging for #WomenWhoCodeSV                                                                         |
-| [Abdou Masoudi](https://github.com/uplancer)       | a selft-taught Web Developer.|
-| [Viraj](https://github.com/viraj-patil)            | I'm a full stack developer from Pune, India. Total 9+ years of experience in software industry as a developer. |
-| [Nitin Kadam](https://github.com/ni3-kdm)            | Hi! I'm a full stack developer from Pune, India. Having 5+ years of experience in web development. |
```
Comment (ni3-kdm) [inline docs/TEAM.md:65]: @Akshay2996 this one is removed. Adding again in 3rd PR.

**Pull request merged**